# Import Data

In [13]:
import pandas as pd

In [17]:
data = pd.read_csv("../desktop/Premier.csv", encoding="latin1")
data.head(15)

club_name       player_name  age            position  \
0   Manchester City             Rodri   23  Defensive Midfield   
1   Manchester City     João Cancelo   25          Right-Back   
2   Manchester City       Pedro Porro   19          Right-Back   
3   Manchester City         Angeliño   22           Left-Back   
4   Manchester City      Zack Steffen   24          Goalkeeper   
5   Manchester City   Ryotaro Meshino   21  Attacking Midfield   
6   Manchester City      Scott Carson   33          Goalkeeper   
7   Manchester City     Jack Harrison   23         Left Winger   
8   Manchester City  Tosin Adarabioyo   22         Centre-Back   
9   Manchester City    Arijanet Muric   21          Goalkeeper   
10  Manchester City    Ante Palaversa   19    Central Midfield   
11  Manchester City      Eric García   18         Centre-Back   
12  Manchester City   Patrick Roberts   23        Right Winger   
13  Manchester City      Mix Diskerud   29    Central Midfield   
14  Manchester City      Luke Brattan   29  Defensive Midfield   

   club_involved_name                      fee transfer_movement  fee_cleaned  \
0    Atlético Madrid                 £63.00m                in         63.0   
1            Juventus                 £58.50m                in         58.5   
2              Girona                 £10.80m                in         10.8   
3       PSV Eindhoven                 £10.80m                in         10.8   
4       Columbus Crew                  £7.20m                in          7.2   
5         Gamba Osaka                   £900k                in          0.9   
6               Derby                     Loan                in          0.0   
7               Leeds  End of loanMay 31, 2020                in          0.0   
8           Blackburn  End of loanMay 31, 2020                in          0.0   
9        Nottm Forest  End of loanMay 31, 2020                in          0.0   
10       Hajduk Split   End of loanJul 2, 2019                in          0.0   
11       Man City U23                        -                in          0.0   
12            Norwich  End of loanMay 31, 2020                in          0.0   
13              Ulsan  End of loanDec 31, 2019                in          0.0   
14     Melbourne City  End of loanJun 30, 2019                in          0.0   

       league_name  year     season  
0   Premier League  2019  2019/2020  
1   Premier League  2019  2019/2020  
2   Premier League  2019  2019/2020  
3   Premier League  2019  2019/2020  
4   Premier League  2019  2019/2020  
5   Premier League  2019  2019/2020  
6   Premier League  2019  2019/2020  
7   Premier League  2019  2019/2020  
8   Premier League  2019  2019/2020  
9   Premier League  2019  2019/2020  
10  Premier League  2019  2019/2020  
11  Premier League  2019  2019/2020  
12  Premier League  2019  2019/2020  
13  Premier League  2019  2019/2020  
14  Premier League  2019  2019/2020

In [18]:
data.shape

(563, 11)

I have a dataset with 563 rows and 11 columns.

# Cleaning the Dataset

In [19]:
data.columns

Index(['club_name', 'player_name', 'age', 'position', 'club_involved_name',
       'fee', 'transfer_movement', 'fee_cleaned', 'league_name', 'year',
       'season'],
      dtype='object')

In [20]:
clean_data(data)

NameError: name 'clean_data' is not defined